### PR TITLE
Create Bolivia parser and config

### PR DESCRIPTION
**Pull request by igorarduin**

### CODE REVIEW

File: parsers/config.py [hunk 1]
```diff
@@ -301,6 +303,7 @@
 
 GENERATION_FORECAST_PARSERS = {
     'DK': ENTSOE.fetch_generation_forecast
```
Comment (corradio) [inline parsers/config.py]: Missing comma at end of line

File: parsers/config.py [hunk 2]
```diff
@@ -65,6 +66,7 @@
     'AT': ENTSOE.fetch_production,
     'BA': ENTSOE.fetch_production,
     'BE': ENTSOE.fetch_production,
+    'B0': BO.fetch_production,
```
Comment (corradio) [inline parsers/config.py]: This should be a O and not a 0 (zero)

File: README.md [hunk 3]
```diff
@@ -122,6 +124,7 @@ Production capacities are centralized in the [capacities.json](https://github.co
 - Germany: [Fraunhoffer](https://energy-charts.de/power_inst.htm)
 - Great Britain: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Greece: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
+- Guatemala: [AMM](http://www.amm.org.gt/pdfs2/2017/Capacidad_Instalada_2017.xls)
```
Comment (corradio) [inline README.md]: Mmh this probably shouldn't be here?

Comment (igorarduin) [inline README.md]: I forgot to add the source when updating Guatemala capacities. I thought I could use this pull request to do it? Or should I do another PR?

Comment (corradio) [inline README.md]: it's ok!

**Comment by igorarduin** [comment]
The zero productions correspond to futur hours (not available yet). But I can define it empty if you prefer.

**Comment by corradio** [comment]
Yes that would be better (thus will be ignored by the system). Thanks!

**Comment by igorarduin** [comment]
Ok I'll do that. Then maybe I should update Guatemala's parser too, as it attributes production to zero for futur dates.

**Comment by corradio** [comment]
would be a good idea. Thanks!!

**Comment by igorarduin** [comment]
Sorry for the delay. Did not have access to a computer for the past week. Tell me if the output fits you better. Then I'll do the same modification on Guatemala's parser.

**Comment by corradio** [comment]
Hey @igorarduin,

Unfortunately I changed the way config and capacities work. They are now centralised in config/zones.json. Can you update your PR accordingly? This should greatly simplify things.

Thanks!

**Comment by corradio** [comment]
Closing this one in favour of #701
